scenarios/centos-9/tcib.yml

```diff
@@ -0,0 +1,6 @@
+cifmw_installyamls_repos: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+ansible_user_dir: "{{ lookup('env', 'HOME') }}"
+cifmw_build_containers_tcib_src: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/tcib"
+cifmw_repo_setup_src: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/repo-setup"
+cifmw_repo_setup_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}"
+cifmw_build_containers_repo_dir: "{{ cifmw_repo_setup_basedir }}/artifacts/repositories"
```

Comment:
We also need to define following this things.
```
cifmw_build_containers_push_registry: localhost:5001
cifmw_build_containers_push_containers: true
```
so that we can push the containers to local registry.